src/components/SeqVizParamsModal.vue

```diff
@@ -214,7 +214,7 @@
                     // Continue with old value
                 }
                 this.integerReminder(
-                    (p.value as BigInt).toString(),
+                    (p.value as bigint).toString(),
                     target.value,
```

Comment:
Thanks for fixing this typo in SeqVizParamsModal.vue. As Liam mentioned, we are not sure how it got there. However, there is one more BigInt that should be bigint, on line 105. Since we are addressing this here, please could you fix that one, too? (lint isn't finding it because it appears to be in a string, from TypeScript's point of view.)